karaf/apache-brooklyn/src/main/resources/etc/org.apache.brooklyn.classrename.cfg

```diff
@@ -0,0 +1,21 @@
+################################################################################
```

Comment:
@neykov @m4rkmckenna I created https://github.com/apache/brooklyn-server/pull/504 in `brooklyn-server` and https://github.com/apache/brooklyn-dist/pull/75 here to do this.

The current implementation will self-extract the files on first boot